internal/jimm/access.go

```diff
@@ -804,3 +804,59 @@ func (j *JIMM) ListGroups(ctx context.Context, user *openfga.User, pagination pa
 	}
 	return groups, nil
 }
+
+// OpenFGACleanup queries OpenFGA for all existing tuples, tries to resolve each tuple and removes those
+// that JIMM cannot resolved - orphaned tuples. JIMM not being able to resolve a tuple means that the
+// corresponding entity has been removed from JIMM's database.
+func (j *JIMM) OpenFGACleanup(ctx context.Context) error {
```

Comment:
Let's update the godoc to make it very explicit that this cleanup method is intended to be short-lived until tuples are correctly removed from OpenFGA. Something like.
```
//....
// 
// This approach to cleaning up tuples is intended to be temporary while we implement
// a better approach to eventual consistency of JIMM's database objects and OpenFGA tuples.
```